streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -329,6 +331,10 @@
     public static final String NUM_STREAM_THREADS_CONFIG = "num.stream.threads";
     private static final String NUM_STREAM_THREADS_DOC = "The number of threads to execute stream processing.";
 
+    /** {@code topology.optimization} */
+    public static final String TOPOLOGY_OPTIMIZATION = "topology.optimization";
```

Comment:
Nit: can we insert this in alphabetic order?

Reply:
ack